src/amo/reducers/reviews.js

```diff
@@ -255,8 +269,10 @@ type ReviewActionType =
   | SetAddonReviewsAction
   | SetInternalReviewAction
   | SetLatestReviewAction
+  | SetRatingSummaryAction
   | SetReviewAction
   | SetReviewReplyAction
+  | SetUserReviewsAction
```

Comment:
🔥

Reply:
This was weird. I did get a Flow error (but not before?). I think the error only popped up when I added a new switch case. Maybe this needed the [`default: (action: empty;)` syntax](https://flow.org/en/docs/react/redux/#toc-typing-redux-reducers) which I never fully understood. We don't use it.